internal/internal_workflow_testsuite.go

```diff
@@ -963,6 +1002,169 @@ func (env *testWorkflowEnvironmentImpl) ExecuteActivity(parameters executeActivi
 	return activityInfo
 }
 
+// Copy of the server function func (v *decisionAttrValidator) validateActivityScheduleAttributes
```

Comment:
We are getting more and more cases when server and SDK shares some low level code (i.e. `serviceerror`). Should't we create some shared library?

Reply:
This is for testing. And I believe we want to share practically the whole service in embedded mode here.